### PR TITLE
Add two-fer

### PR DESCRIPTION
**Pull request by BNAndras**
Related to #172

### CODE REVIEW

**Review by kotp — APPROVED** [review]

**Comment by BNAndras** [comment]
Looks like track config needs to be updated. 
```"example": [
      "example.vim"
    ],
```

should be ``` "example": [
      ".meta/example.vim"
    ],
```

**Comment by BNAndras** [comment]
Track config update is at #175

File: exercises/practice/two-fer/two_fer.vim [hunk 1]
```diff
@@ -10,5 +10,5 @@
 "" One for Alice, one for me.
 ""
 function! TwoFer(name) abort
-  "your code goes here"
+  " your code goes here"
```
Comment (kotp) [inline exercises/practice/two-fer/two_fer.vim:13]: Why a space here?

Comment (kotp) [reply exercises/practice/two-fer/two_fer.vim:13]: Also, we could probably set it as the example, `"One for Alice, one for me."`

Comment (BNAndras) [reply exercises/practice/two-fer/two_fer.vim:13]: There's a space between the double quotes and the message to implement the function for the other exercises I checked so I added it for consistency.

Comment (kotp) [reply exercises/practice/two-fer/two_fer.vim:13]: Doh.  Yep, comment marker, not string delimiter.

Comment (kotp) [reply exercises/practice/two-fer/two_fer.vim:13]: Should we use the closing quotes then?

Comment (kotp) [reply exercises/practice/two-fer/two_fer.vim:13]: Yeah, just double checked, it is not terminated with another comment marker for at least the first few exercises.

File: exercises/practice/two-fer/two_fer.vim [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+"" Example:
+""
+"" :echo TwoFer('')
+"" One for you, one for me.
+""
+"" :echo TwoFer('Alice')
+"" One for Alice, one for me.
+""
+function! TwoFer(name) abort
+  " your code goes here"
```
Comment (kotp) [inline exercises/practice/two-fer/two_fer.vim:13]:
```suggestion
  " your code goes here
```

**Review by kotp — CHANGES_REQUESTED** [review]

**Review by kotp — APPROVED** [review]

**Pull request merged**